R30.s

```diff
@@ -340,7 +340,7 @@ SPLRET		EXIT
 		EXTEND
 		BZF	SPLRET1
 V82GON3		CAF	BIT5
-		MASK	EXTVBACT	# SEE IF ASTRONAUT HAS SIGNALLED TERMINATE
+		MASK	EXTVBACT	# SEE IF ASTRONAUT HAS SIGNALED TERMINATE
```

Comment:
`SIGNALLED` is used in [source](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/0520.jpg)